src/core/server/http/router/socket.test.ts

```diff
@@ -92,7 +92,7 @@ describe('KibanaSocket', () => {
       });
       const socket = new KibanaSocket(tlsSocket);
 
-      expect(socket.renegotiate({})).resolves.toBe(result);
+      await expect(socket.renegotiate({})).rejects.toBe(result);
```

Comment:
because we call `callback` with a non-null argument in line 90 this promise rejects